tests/xharness/Jenkins.cs

```diff
@@ -1367,7 +1368,8 @@ void GenerateReportImpl (Stream stream, StreamWriter markdown_summary = null)
 					markdown_summary.Write (string.Join (", ", list));
 					markdown_summary.WriteLine ();
 				} else if (failedTests.Any ()) {
-					markdown_summary.WriteLine ($"{failedTests.Count ()} tests failed, {passedTests.Count ()} tests passed.");
+					var skipped = skippedTests.Any () ? $" and {skippedTests.Count ()} tests skipped" : "";
+					markdown_summary.WriteLine ($"{failedTests.Count ()} tests failed, {passedTests.Count ()} tests passed{skipped}.");
```

Comment:
This won't be printed if there are only skipped tests, in which case it'll just show "All ### tests passed" (the next if just below), which is wrong. I think you need to change the condition above to:

```csharp
} else if (failedTests.Any () || skippedTests.Any ()) {
```